client/my-sites/site-settings/spam-filtering-settings/index.jsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { isJetpackModuleActive } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { updateSettings } from 'state/jetpack/settings/actions';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+const SpamFilteringSettings = ( {
+	isRequestingSettings,
+	isSavingSettings,
+	akismetActive,
+	translate
+} ) => {
+	const header = akismetActive
+		? translate( 'Your site is protected from spam' )
+		: translate( 'Install & activate Spam filtering' );
+	return (
+		<div>
+			<FoldableCard header={ header }>
+				<div className="spam-filtering-settings__module-settings">
+						<div className="spam-filtering-settings__info-link-container site-settings__info-link-container">
+							<InfoPopover >
+								<ExternalLink target="_blank"
+									icon={ true }
+									href={ 'https://jetpack.com/features/security/spam-filtering/' }
+								>
+									{ translate( 'Learn more about Spam Filtering.' ) }
+								</ExternalLink>
+							</InfoPopover>
+						</div>
+					<FormFieldset>
+						<FormLabel>{ translate( 'Your API Key ' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Get your API key' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/akismet-card-for-jetpack-sites-in-settings-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).